= Neo4j Life Sciences and Healthcare Network
:section: Use Cases
:section-link: use-cases
:section-level: 2
:slug: life-sciences-and-healthcare
:toc:
:toc-placement!:
:toc-title: Overview
:toclevels: 2

image:{img}icon-guide-theory.png[float=left]

== Neo4j Use Cases in Life Sciences and Healthcare

If you work in biology, biochemistry, pharmaceuticals, healthcare and other life sciences, you know that you work with highly-connected information. Unfortunately, many scientists still use relational databases and spreadsheets as their daily tools.


Here we want to present you with an alternative. Managing, storing and querying connected information is natural to a graph database like Neo4j. Learn how your research and practitioner colleagues utilized Neo4j to draw new insights or just be more efficient in their daily work.


It started a while time ago in 2012 with a http://neo4j.com/blog/graph-databases-in-life-sciences-workshop/[workshop at the University of Ghent] bringing together people from the field with graph database experts.

Following that fruitful exchange we started the http://groups.google.com/group/neo4j-biotech[Neo4j-Biotech Google Group] to encourage sharing and collaboration on that topic. If you are not yet a member, please join today.

Now we want to take it to the next level by providing you with a platform to present your projects and papers both here and on our blog, and giving you the opportunity to connect with other Neo4j users in your field.


If you are taking your first steps towards using a graph database, we offer support to jumpstart your efforts.
// a dedicated Accelerator Program

////
Create a good offering to help them get started, problem solving etc
Incentive to reach out to us
Link to papers
Publication of blog posts on neo4j.com
benefits of using a graph database over e.g. a relational databases
examples e.g. for data models
Note email group, neo4j-contact
Talk about the planned workshop for 2017
////

toc::[]

=== Why use a Graph Database

++++
<iframe width="560" height="315" src="https://www.youtube.com/embed/V7f2tGsNSck?showinfo=0&controls=2&autohide=1" frameborder="0" allowfullscreen></iframe>
++++


// === How to share your work/paper

=== Graph Databases in Life and Health Sciences Workshop: Berlin, 21 June 2017

We are very pleased to announce our second workshop for researchers interested in sharing and learning about Graph Databases in Life and Health Sciences.

We are inviting researchers, practitioners and developers to present and attend.

More details, as well as registration information, can be found https://www.eventbrite.com/e/neo4j-life-health-sciences-day-berlin-tickets-33238223421[here].


// If you want to participate or submit a talk or paper, please let us know via devrel@neo4j.com.


In our http://neo4j.com/blog/graph-databases-in-life-sciences-workshop/[past workshop in Ghent], we had topics covering

* Neo4j in metaproteonomics
* Graph databases in cancer research
* Project collaboration networks and recommendations
* Detailed studies of citation graphs
* Connecting protein databases in a large graph model
* "Reactome" database of human protein interaction pathways

=== Life Sciences and Healthcare Accelerator Program


The Neo4j Life Sciences and Healthcare Accelerator Program is designed to help researchers and practitioners in life sciences and healthcare-related sciences make sense of their data using Neo4j. Whether you are analyzing genome data, combining protein databases, investigating drug interactions or supporting practitioners with research or clinical information processing we want to help you find insights in connected (meta-)data.


If you are accepted into the program, you will receive 1-on-1 assistance from Neo4j engineers to help you with data modeling, data import, writing Cypher queries or anything else that we can to make you successful with Neo4j.


To get started just tell us about your project and how you think we might be able to help you.


https://goo.gl/forms/T9wkIZVms1XE7kNE3[Apply Here,role="button apply"]


=== Featured Projects


==== The Hetnet Awakens - Understanding Disease Through Data Integration and Open Science

_Daniel Himmelstein_

Daniel Himmelsteinâ€™s Thesis Seminar for his PhD in Biological & Medical Informatics at UCSF.

Here are the https://slides.com/dhimmel/thesis-seminar[slides] and an http://blog.dhimmel.com/phd-exhibit[online adaptation of the PhD Exhibit].
Daniel was also interviewed on our http://blog.bruggen.com/2016/08/podcast-interview-with-daniel.html[Graphistania Podcast] and created a fun http://neo4j.com/graphgist/c4eab62c-7f5e-4e17-8f75-811d65d83127[Graph Gist as live documentation].

++++
<iframe width="560" height="315" src="https://www.youtube.com/embed/H8DfXop8K7g" frameborder="0" allowfullscreen></iframe>
++++


==== Proteomics and Graph Databases, the symbiosis of associations

_Alejandro Brenes Murillo_

The proteome is the entire set of proteins that are produced or modified by an organism. It is an element that varies with time, stress, environmental conditions or distinct requirements that a cell might have. Join this talk by Alejandro Brenes Murillo to see how graph databases can be useful for proteome analysis.


At the Lamond Lab in the University of Dundee, scientists are interested in modelling and understanding protein behaviour under different conditions and dimensions of analysis.


In order to achieve this goal, they use graph databases to integrate and model the proteomics data, and study its effect on a specific proteome. The dimensions of analysis are multiple, yet be it turnover, localisation, cell cycle, protein complexes or biological response to stimuli, discovering the behaviour of proteins is key to understanding how organisms function, and how disease affects them.

image::{img}/alejandro_proteomics.jpg[link="https://skillsmatter.com/skillscasts/9246-proteomics-and-graph-databases-the-symbiosis-of-associations",width=560,window="_blank"]


==== Big Data in Genomics: How Neo4j helps to develop new drugs

_Martin Preusse_

Biomedical research generates vast amounts of data. New experimental technologies like DNA sequencing, metabolomics and proteomics drive the fast growth of available information and lead to a better understanding of the molecular organization of life.


But with big data comes a big question: How do we transform unstructured data into actionable knowledge? In the case of biomedical research, the key problem is to integrate the large pile of highly heterogenous data and use it for personalized therapies and drug development. Graph databases are an ideal way to represent biomedical knowledge and offer the necessary flexibility to keep up with scientific progress. A well-designed data model and Cypher queries can deliver in seconds what previously took days of manual analysis.

image::{img}/preusse_genomics.jpg[link="https://skillsmatter.com/skillscasts/7302-big-data-in-genomics-how-neo4j-helps-to-develop-new-drugs",width=560,window="_blank"]

==== Building a Repository of Biomedical Ontologies with Neo4j

_Simon Jupp_

In this lightning talk from GraphConnect Europe 2016, Simon Jupp of the European Bioinformatics Institute discusses the application they built to track ontologies. He also discusses why they chose Neo4j over various RDF and semantic web technologies, and provides some example queries.

++++
<iframe width="560" height="315" src="https://www.youtube.com/embed/jZND2WYT4GE" frameborder="0" allowfullscreen></iframe>
++++

==== Data Management in Systems Biology & Medicine

_Irina Balaur, EISBM_

An Integrative Framework for Data Management in Systems Biology and Medicine: Strategies for personalised medicine involve integration of large amounts of biomedical data, specific to multiple spatial and temporal scales, (including molecular data and patient clinical data). We have been developing a graph-database approach implemented in Neo4j to facilitate management (integration, exploration, visualisation, interpretation) of diverse types of biological and biomedical data.


++++
<iframe width="560" height="315" src="https://www.youtube.com/embed/ukvVVMhJACE" frameborder="0" allowfullscreen></iframe>
++++

====  Graphs Are Feeding The World

_Tim Williamson, Data Scientist, Monsanto_

Presentation at GraphConnect SF 2015.

++++
<iframe width="560" height="315" src="https://www.youtube.com/embed/6KEvLURBenM" frameborder="0" allowfullscreen></iframe>
++++


==== Graph Databases in Life Sciences: Bringing Biology Back to Its Nature

_Thilo Muth_


Today's life science research is about genes, proteins, metabolites, relationships, interactions and biological networks. Data storing and mining brings a huge potential for biologists, however classical storage formats such as SQL and Excel involve various issues, such as scalability and performance problems with data growth, complexity and accessibility. Finally, most of the storage models are far from biological reality: Graph databases and Neo4j meet the need in life sciences for an appropriate data and database model.

++++
<iframe src="https://player.vimeo.com/video/77450061" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
++++


==== Open Tree Of Life

image::http://opentreeoflife.files.wordpress.com/2012/04/opentree-final-logo-copy.png?w=360&h=189[float="left"]

*The tree of life* links all biodiversity through a shared evolutionary history.
This project will produce the first online, comprehensive first-draft tree of all 1.8 million named species, accessible to both the public and scientific communities.

Assembly of the tree will incorporate previously-published results, with strong collaborations between computational and empirical biologists to develop, test and improve methods of data synthesis.

This initial tree of life will not be static; instead, we will develop tools for scientists to update and revise the tree as new data come in.
Early release of the tree and tools will motivate data sharing and facilitate ongoing synthesis of knowledge.


*Biological research* of all kinds, including studies of ecological health, environmental change, and human disease, increasingly depends on knowing how species are related to each other.

Yet there is no single resource that unites knowledge of the tree of life.
Instead, only small parts of the tree are individually available, generally as printed figures in journal articles.

This project will provide the global community of scientists who study the tree of life with a means to share and combine their results, and will enable large-scale studies of Earthâ€™s biodiversity.
It will also create a resource where students, educators and citizens can go to explore and learn about lifeâ€™s evolutionary history.

Read more on the http://blog.opentreeoflife.org/tag/neo4j/[OpenTreeOfLife Blog]

++++
<iframe src="https://player.vimeo.com/video/67870035" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
<p><a href="https://vimeo.com/67870035">0606 - Open Tree of Life and Neo4j</a> from <a href="https://vimeo.com/neo4j">Neo Technology</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
++++


////
=== Project Catalogue


[cols="2,1,3",opts="header"]
|===
| Name
| Contact
| Description


| http://knowing-health.com/[Startup: KNOWING â€“ Turning Big Data into Personalized Therapies]
| Martin Preusse
| Developing a software that can be used for result integration of any type of research data as well as be used to harvest data to identify biomarker and drug targets for personalized therapies.

|===

////

=== Publications

[[publications]]
[cols="3,1,3,3",options="header"]
|===
| Title
| Year
| Authors
| Affiliation

| https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx237[The Proteins API: accessing key integrated protein and genome information]
| 2017
| A. Nightingale, R. Antunes, E. Alpi, B. Bursteinas, L. Gonzales, W. Liu, J. Luo, G. Qi, E. Turner, and M. Martin
| EMBL-EBI, Wellcome Genome Campus, UK

| http://biorxiv.org/content/biorxiv/early/2016/05/26/055525.full.pdf[Knowledge.Bio: A Web application for exploring, building and sharing webs of biomedical relationships mined from PubMed]
| 2016
| R. Bruskiewich, K. Huellas-Bruskiewicz, F. Ahmed, R. Kaliyaperumal, M. Thompson, E. Schultes, K. M. Hettne, A. I. Su, and B. M. Good
| Department of Human Genetics, Leiden University Medical Center, The Netherlands

| https://academic.oup.com/bioinformatics/article/doi/10.1093/bioinformatics/btw731/2557691/Recon2Neo4j-applying-graph-database-technologies[Recon2Neo4j: Applying graph database technologies for managing comprehensive genome-scale networks]
| 2016
| I. Balaur, A. Mazein, M. Saqi, A. Lysenko, C. J. Rawlings, and C. Auffray
| European Institute for Systems Biology and Medicine (EISBM), France

| https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1394-x[STON: exploring biological pathways using the SBGN standard and graph databases]
| 2016
| V. TourÃ©, A. Mazein, D. Waltemath, I. Balaur, M. Saqi, R. Henkel, J. Pellet, and C. Auffray
| European Institute for Systems Biology and Medicine (EISBM), France

| https://www.ncbi.nlm.nih.gov/pubmed/26998997[miTALOS v2: Analyzing Tissue Specific microRNA Function]
| 2016
| M. Preusse, F. J. Theis, and N. S. Mueller
| Institute of Computational Biology, Helmholtz Zentrum MÃ¼nchen, Germany

| https://www.ncbi.nlm.nih.gov/pubmed/27196054[An Integrated Data Driven Approach to Drug Repositioning Using Gene-Disease Associations]
| 2016
| J. Mullen, S. J. Cockell, P. Woollard, and A. Wipat
| Newcastle University, United Kingdom

| https://www.ncbi.nlm.nih.gov/pubmed/26708334[HitWalker2: visual analytics for precision medicine and beyond]
| 2016
| D. Bottomly, S. K. McWeeney, and B. Wilmot
| Knight Cancer Institute, Oregon Health and Science University, USA

| https://www.ncbi.nlm.nih.gov/pubmed/26657893[HRGRN: A Graph Search-Empowered Integrative Database of Arabidopsis Signaling Transduction, Metabolism and Gene Regulation Networks]
| 2016
| X. Dai, J. Li, T. Liu, and P. X. Zhao
| Plant Biology Division, The Samuel Roberts Noble Foundation, USA

| https://www.ncbi.nlm.nih.gov/pubmed/27462371[Representing and querying disease networks using graph databases]
| 2016
| A. Lysenko, I. A. RoznovÄƒÅ£, M. Saqi, A. Mazein, C. J. Rawlings, and C. Auffray
| European Institute for Systems Biology and Medicine (EISBM), France

| https://www.ncbi.nlm.nih.gov/pubmed/27587666[PanTools: representation, storage and exploration of pan-genomic data]
| 2016
| S. Sheikhizadeh, M. E. Schranz, M. Akdel, D. de Ridder, and S. Smit
| Bioinformatics Group, Wageningen University, The Netherlands

| https://www.ncbi.nlm.nih.gov/pubmed/27627442[EpiGeNet: A Graph Database of Interdependencies Between Genetic and Epigenetic Events in Colorectal Cancer]
| 2016
| I. Balaur, M. Saqi, A. Barat, A. Lysenko, A. Mazein, C. J. Rawlings, H. J. Ruskin, and C. Auffray
| European Institute for Systems Biology and Medicine (EISBM), France

| http://www.ncbi.nlm.nih.gov/pubmed/26272981[cyNeo4j: connecting Neo4j and Cytoscape]
| 2015
| G. Summer, T. Kelder, K. Ono, M. Radonjic, S. Heymans, and B. Demchak
| Center for Heart Failure Research (CARIM), University Hospital Maastricht, The Netherlands

| https://www.thinkmind.org/download.php?articleid=dbkda_2015_9_50_90009[Towards Implementing Semantic Literature-Based Discovery with a Graph Database]
| 2015
| D. Hristovski, A. Kastrin, D. Dinevski, and T. C. Rindflesch
| Faculty of Medicine, University of Ljubljana, Slovenia

| http://dl.acm.org/citation.cfm?id=2918100[Using Neo4j for Mining Protein Graphs: A Case Study]
| 2015
| D. Hoksza and J. Jelinek
| Faculty of Mathematics and Physics, Charles University in Prague, Czech Republic

| http://pubs.acs.org/doi/abs/10.1021/pr501246w[The MetaProteomeAnalyzer: A Powerful Open-Source Software Suite for Metaproteomics Data Analysis and Interpretation]
| 2015
| T. Muth, A. Behne, R. Heyer, F. Kohrs, D. Benndorf, M. Hoffmann, M. LehtevÃ¤, U. Reichl, L. Martens, and E. Rapp
| Max Planck Institute for Dynamics of Complex Technical Systems, Magdeburg, Germany

| https://www.ncbi.nlm.nih.gov/pubmed/26383775[SimiRa: A tool to identify coregulation between microRNAs and RNA-binding proteins]
| 2015
| M. Preusse, C. Marr, S. Saunders, D. Maticzka, H. Lickert, R. Backofen, and F. Theis
| Helmholtz Zentrum MÃ¼nchen, Institute of Computational Biology, Germany

| https://www.ncbi.nlm.nih.gov/pubmed/26262393[Constructing a Graph Database for Semantic Literature-Based Discovery]
| 2015
| D. Hristovski, A. Kastrin, D. Dinevski, and T. C. Rindflesch
| Faculty of Medicine, University of Ljubljana, Slovenia

| https://www.ncbi.nlm.nih.gov/pubmed/25708381[A systems biology approach toward understanding seed composition in soybean]
| 2015
| L. Li, M. Hur, J. Y. Lee, W. Zhou, Z. Song, N. Ransom, C. Y. Demirkale, D. Nettleton, M. Westgate, Z. Arendsee, V. Iyer, J. Shanks, B. Nikolau, and E. S. Wurtele
| Department of Genetics, Development and Cell Biology, Iowa State University, USA

| https://www.ncbi.nlm.nih.gov/pubmed/25754863[Combining computational models, semantic annotations and simulation experiments in a graph database]
| 2015
| R. Henkel, O. Wolkenhauer, and D. Waltemath
| Department of Computer Science, University of Rostock, Germany

| https://www.ncbi.nlm.nih.gov/pubmed/26305513[An alternative database approach for management of SNOMED CT and improved patient data queries]
| 2015
| W. S. Campbell, J. Pedersen, J. C. McClay, P. Rao, D. Bastola, and J. R. Campbell
| University of Nebraska Medical Center, Department of Pathology and Microbiology, US

| https://www.ncbi.nlm.nih.gov/pubmed/25520553[Semantically linking in silico cancer models]
| 2014
| D. Johnson, A. J. Connor, S. McKeever, Z. Wang, T. S. Deisboeck, T. Quaiser, and E. Shochat
| Department of Computing, Imperial College London, London, UK

| http://www.sciencedirect.com/science/article/pii/S1574954114001125[Global biotic interactions: An open infrastructure to share and analyze species-interaction datasets]
| 2014
| J. H. Poelen, J. D. Simons, and C. J. Mungall
| Center for Coastal Studies Natural Resource Center, USA

| http://bioinformatics.oxfordjournals.org/content/early/2013/10/21/bioinformatics.btt549.full[Are graph databases ready for bioinformatics?]
| 2013
| Christian Theil Have and Lars Juhl Jensen
| Department of Metabolic Genetics, University of Copenhagen, Denmark

|===

++++
<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css">
<script src="https://code.jquery.com/jquery-1.12.4.js"></script>
<script src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
<script>
$(document).ready(function() {
    $('#publications').DataTable();
} );
</script>
++++